src/ref/builtin.md

```diff
@@ -1393,6 +1393,26 @@ input elements, the entire input is retained. Examples:
 Etymology: Haskell.
 
 
+## tilde-abbr
+
+```elvish
+tilde-abbr $pwd
```

Comment:
s/$pwd/$path/. This argument is not necessarily the pwd, not even has to be a directory.